pkg/expr/nodes.go

```diff
@@ -401,7 +403,44 @@ func (dn *DSNode) Execute(ctx context.Context, now time.Time, _ mathexp.Vars, s
 	}
 
 	var result mathexp.Results
-	responseType, result, err = s.converter.Convert(ctx, dn.datasource.Type, dataFrames, s.allowLongFrames)
+	// TODO: Don't run Datasource Node Results through converter if input is SQL expression
+	// Also for now, must be one frame and not of the fields may have labels
+	if dn.noConvert {
+		var convertForSQL bool
+		if len(dataFrames) > 1 {
+			convertForSQL = true
+		}
+
+		for _, frame := range dataFrames {
+			if convertForSQL {
+				break
+			}
+			for _, field := range frame.Fields {
+				if field.Labels != nil {
+					convertForSQL = true
+					break
+				}
+			}
+		}
+
+		if convertForSQL {
```

Comment:
I think we will want SQL conversion "later" in processing (near the input of SQL). Then I think might be easier to get it working with other SSE operations as input.